go/grpcwebproxy/main.go

```diff
@@ -151,3 +159,51 @@ func buildListenerOrFail(name string, port int) net.Listener {
 		conntrack.TrackWithTracing(),
 	)
 }
+
+func makeHttpOriginFunc(allowedOrigins *allowedOrigins) func(origin string) bool {
+	if *flagAllowAllOrigins {
+		return func(origin string) bool {
+			return true
+		}
+	} else {
+		return func(origin string) bool {
```

Comment:
This could just be `return allowedOrigins.IsAllowed`.

Reply:
Dog!